Cargo.toml

```diff
@@ -14,4 +14,4 @@ license = "MIT"
 strict = []
 
 [dependencies]
-bech32 = "0.7.1"
+bech32 = "0.7.3"
```

Comment:
Needs a bump to at least 0.8.0, maybe we also need a new release to add some derives there? Or maybe that was already included in 0.8.0?

Reply:
New derives went in before 0.8.0

Reply:
Still, having 0.8 will be better, or not?

Reply:
That's right. We depend on 0.8.0 for the Bech32m support from #31 .